tools/metadata.json

```
@@ -1,9 +1,9 @@
 {
   "StableReleaseTag": "v7.0.3",
-  "PreviewReleaseTag": "v7.1.0-preview.6",
+  "PreviewReleaseTag": "v7.1.0-preview.7",
   "ServicingReleaseTag": "v6.2.7",
   "ReleaseTag": "v7.0.3",
   "LTSReleaseTag" : ["v7.0.3"],
-  "NextReleaseTag": "v7.1.0-preview.7",
+  "NextReleaseTag": "v7.1.0-rc.1",
```

Keep this is a preview, even if it won't be
```suggestion
  "NextReleaseTag": "v7.1.0-preview.8",
```